backend/hdf5/SectionHDF5.cpp

```diff
@@ -251,24 +251,32 @@ shared_ptr<IProperty> SectionHDF5::getProperty(ndsize_t index) const {
 }
 
 
-shared_ptr<IProperty> SectionHDF5::createProperty(const string &name, const DataType &dtype) {
+shared_ptr<IProperty> SectionHDF5::createProperty(const string &name, const DataType &dtype, const NDSize &shape) {
     string new_id = util::createId();
     boost::optional<H5Group> g = property_group(true);
-    DataSet ds = g->createData(name, data_type_to_h5_filetype(dtype), {0});
+    DataSet ds = g->createData(name, data_type_to_h5_filetype(dtype), shape, Compression::DeflateNormal,
+                               {}, shape, true, false);
     return make_shared<PropertyHDF5>(file(), ds, new_id, name);
 }
 
 
+shared_ptr<IProperty> SectionHDF5::createProperty(const string &name, const DataType &dtype) {
+    shared_ptr<IProperty> p = createProperty(name, dtype, {8});
```

Comment:
why `8`? I would suggest a define, `DEFAULT_PROPERTY_SIZE`, with some comment about the trade offs,

Reply:
haha, fair enough, will do